src/components/WebCLI/Output.tsx

```diff
@@ -71,20 +80,21 @@ const DEFAULT_HEIGHT = 300;
 // an inoportune time and the element isn't left completely closed.
 const CONSIDER_CLOSED = 20;
 const HELP_HEIGHT = 50;
+const dragHandles = ["webcli__output-dragarea", "webcli__output-handle"];
 
 const WebCLIOutput = ({
   content,
   helpMessage,
   showHelp,
   setShouldShowHelp,
-}) => {
+}: Props) => {
```

Comment:
From here and above are the TypeScript changes, below this are the fixes for resizing the CLI.